test/helpers/position_assertions.cc

```diff
@@ -97,6 +97,85 @@ string uriToFilePath(string_view prefixUrl, string_view uri) {
     return string(uri.substr(prefixUrl.length() + 1));
 }
 
+void assertLocationsMatch(const UnorderedMap<string, shared_ptr<core::File>> &sourceFileContents, string_view uriPrefix,
```

Comment:
Both lists have to be sorted for this to work. Should we move `fast_sort` here? It would mean not taking a const reference, but I think it removes that (somewhat confusing) responsibility off of the callers.

Reply:
Yep, that's a good suggestion. I think i'm going to fast-follow with this update so i don't have to do a song and dance of rebasing.

Reply:
I ended up folding this into #1704